docs/spec.md

```diff
@@ -0,0 +1,316 @@
+   </td>
+   <td>Yes
+   </td>
+   <td>A unique ID for this node
+   </td>
+  </tr>
+  <tr>
+   <td>StatementType
+   </td>
+   <td>Yes
+   </td>
+   <td>Must be "personStatement" to generate person node
+   </td>
+  </tr>
+  <tr>
+   <td>names
+   </td>
+   <td>No
+   </td>
+   <td>Unknown person will be used if this data is missing. The name types will be used in the following order: 'individual', 'transliteration', 'alternative', 'birth', 'translation', 'former'. Unnamed person will be used if any specific name data is missing.
+   </td>
+  </tr>
+  <tr>
+   <td>names.fullName
+   </td>
+   <td>No
+   </td>
+   <td>This field takes precedence
+   </td>
+  </tr>
+  <tr>
+   <td>names.givenName
+   </td>
+   <td>No
+   </td>
+   <td>The name is a join of [givenName, patronymicName, familyName] if fullName is omitted
+   </td>
+  </tr>
+  <tr>
+   <td>names.patronymicName
+   </td>
+   <td>No
+   </td>
+   <td>The name is a join of [givenName, patronymicName, familyName] if fullName is omitted
+   </td>
+  </tr>
+  <tr>
+   <td>names.familyName
+   </td>
+   <td>No
+   </td>
+   <td>The name is a join of [givenName, patronymicName, familyName] if fullName is omitted
+   </td>
+  </tr>
+  <tr>
+   <td>personType
+   </td>
+   <td>No
+   </td>
+   <td>Unknown person will be assumed if the personType is missing.
+   </td>
+  </tr>
+  <tr>
+   <td>nationalities
+   </td>
+   <td>No
+   </td>
+   <td>No flags will be drawn if these data are missing. The visualiser only uses the first nationality in the nationalities array.
+   </td>
+  </tr>
+  <tr>
+   <td>nationalities.code
+   </td>
+   <td>No
+   </td>
+   <td>No flag will be drawn if the country code is missing. The visualiser only uses the first nationality in the nationalities array.
+   </td>
+  </tr>
+</table>
+
+
+
+### Entity Nodes
+Suggest adding 'Entity nodes are drawn using BODS (or BODS-like) [entity statements](https://standard.openownership.org/en/0.3.0/schema/schema-browser.html#entity-statement).
+
+<table>
+  <tr>
+   <td><strong>Field</strong>
+   </td>
+   <td><strong>Required</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>statementID
+   </td>
+   <td>Yes
+   </td>
+   <td>Unique ID for this node
+   </td>
+  </tr>
+  <tr>
+   <td>statementType
+   </td>
+   <td>Yes
+   </td>
+   <td>Must be 'entityStatement' for entity node
+   </td>
+  </tr>
+  <tr>
+   <td>name
+   </td>
+   <td>No
+   </td>
+   <td>No label shown if missing
+   </td>
+  </tr>
+  <tr>
+   <td>entityType
+   </td>
+   <td>No
+   </td>
+   <td>Assumes unknown node type when entityType is missing
+   </td>
+  </tr>
+  <tr>
+   <td>publicListing
+   </td>
+   <td>No
+   </td>
+   <td>Assumes entity is not listed if this field is missing
+   </td>
+  </tr>
+  <tr>
+   <td>incorporatedInJurisdiction (BODS v0.2)/jurisdiction (BODS v0.3)
+   </td>
+   <td>No
+   </td>
+   <td>No flags are drawn if this field is missing.
+   </td>
+  </tr>
+  <tr>
+   <td>jurisdiction.code
```

Comment:
I kept this as just BODS v0.3 for brevity, and hopefully the row above makes it clear how to adapt for v0.2 but happy to change if we need greater clarity.